.github/workflows/root-ci-config/build_root.py

```diff
@@ -243,18 +249,15 @@ def show_node_state(shell_log: str, options: str) -> str:
 
     return shell_log
 
-
+#Even in case of some failed tests I am still making it run
```

Comment:
What's "it"? Maybe we remove the comment? Or reword:
```suggestion
# Just return the number of test failures instead of `die()`-ing; report test
# failures later. Coverage can be calculated also with some failing tests.
```